index.html

```diff
@@ -37,90 +37,75 @@
 	<section data-background-color="#ffffff">
 	  <table class="reveal">
 		<tr>
-		<a target="_blank" href="https://mcin.ca"><img width="30%" src="images/MCIN.png"</a>
+		<a href="MCIN.html"><img width="40%" src="images/MCIN.png" style="border: 0"</a>
 		</tr>
+		
+		<tr>
+		<p style="font-size:55%; color:#224372;"><a target="_blank" href="https://mcin.ca/">The McGill Centre for Integrative Neuroscience (MCIN)</a>, led by <a target="_blank" href="https://mcin.ca/about-mcin/alans-cv/">Dr. Alan Evans</a> 
+		at McGill University, creates open neuroinformatics platforms to conduct computationally-intensive brain research using innovative mathematical and statistical approaches to integrate clinical, psychological or neuroimaging phenotypes with genotypic information. 
```

Comment:
```suggestion
		at McGill University, <br> creates open neuroinformatics platforms to conduct computationally-intensive brain research using innovative mathematical and statistical approaches to integrate clinical, psychological or neuroimaging phenotypes with genotypic information. 
```
@iremkatitas @GHPBZ how do we feel about forcing line breaks?  there should also be one before "includes" so that CONP is printed on one line instead of split awkwardly.  
This definitely changes with browsers but i think these 2 line breaks will make sense generally.

Reply:
line breaks before "creates" and "including" added on latest commit

Reply:
great, approving these changes  (and assuming you have looked to see it renders ok)